protos/bottle/account/v1/user.proto

```diff
@@ -2,10 +2,24 @@ syntax = "proto3";
 
 package bottle.account.v1;
 
+import "bottle/account/v1/organization.proto";
+
 message User {
   string id = 1;
   string first_name = 2;
   string last_name = 3;
   string phone_number = 4;
   string email = 5;
+  bool newsletter = 6;
+
+  enum AccountType {
+    ACCOUNT_TYPE_UNSPECIFIED = 0;
+    ACCOUNT_TYPE_INDIVIDUAL = 1;
+    ACCOUNT_TYPE_BUSINESS = 2;
+    ACCOUNT_TYPE_RESELLER = 3;
+  }
+
+  AccountType account_type = 7;
+  string company_name = 8;
+  bottle.account.v1.Organization organization = 9;
```

Comment:
With the other organization changes, this may not be necessary at the moment :+1:

Reply:
I'll remove this from now. Since we don't yet support multiple orgs in Zendesk, we won't need to spend too much time supporting that in our system. Removing this now gives us flexibility later :+1: